src/retention/applier/downtime.cc

```diff
@@ -41,9 +42,6 @@ void applier::downtime::apply(list_downtime const& lst) {
     else
       _add_service_downtime(**it);
   }
-
-  // Sort all downtimes.
-  sort_downtime();
```

Comment:
You do not need this anymore because now the list_downtime is sorted if I am right ?

Reply:
the multimap is a binary tree and is sorted by its key.